### ISSUE TITLE
Can't remove URLPath from admin

### ISSUE THREAD

**Issue by nicoekkart** [django-wiki/django-wiki#668]
When I try to delete a URLPath via the admin, I get the ValueError error "Cannot call get_ancestors on unsaved URLPath instances".

The url is removed after this error, but this prohibits batch removing of url's, since it instantly stops because of the error..

**Comment by benjaoming** [comment]
Thanks for the report @nicoekkart - Are you going to work on this? Could you post the full traceback?

**Comment by nicoekkart** [comment]
I really don't have any time for it right now. 
Anyways, here is the full traceback:
 ```Environment:


Request Method: POST
Request URL: http://localhost:8000/admin/wiki/urlpath/

Django Version: 1.10.3
Python Version: 3.5.2
Installed Applications:
('django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.sites',
 'django.contrib.admin',
 'django.contrib.redirects',
 'django.contrib.flatpages',
 'django.contrib.staticfiles',
 'django.contrib.humanize',
 'VTK.apps.user',
 'VTK.apps.tools',
 'VTK.apps.tools.financetools',
 'VTK.apps.tools.financetools.internal',
 'VTK.apps.tools.financetools.external',
 'VTK.apps.activities',
 'VTK.apps.site_main',
 'VTK.apps.members',
 'VTK.apps.links',
 'VTK.apps.about',
 'VTK.apps.sport',
 'VTK.apps.shiften',
 'VTK.apps.common',
 'VTK.apps.tools.praesidiumtools',
 'VTK.apps.tools.praesidiumtools.statistics',
 'VTK.apps.tools.praesidiumtools.mailer',
 'VTK.apps.tools.praesidiumtools.scantool',
 'VTK.apps.tools.praesidiumtools.volunteers',
 'VTK.apps.tools.praesidiumtools.yolofood',
 'VTK.apps.tools.cursustools',
 'VTK.apps.tools.cafetools',
 'VTK.apps.cursus',
 'VTK.apps.mail',
 'VTK.apps.photos',
 'VTK.apps.firnale',
 'VTK.apps.onthaal',
 'VTK.apps.thermilan',
 'VTK.apps.fris',
 'VTK.apps.centralauth',
 'VTK.apps.ads',
 'VTK.apps.elections',
 'VTK.apps.cafe',
 'VTK.apps.survey',
 'VTK.apps.career',
 'VTK.apps.career.students',
 'VTK.apps.career.companies',
 'VTK.apps.career.services',
 'VTK.apps.jaardrink',
 'VTK.apps.updatewiki',
 'photologue',
 'sortedm2m',
 'djmail',
 'captcha',
 'formtools',
 'widget_tweaks',
 'import_export',
 'nested_admin',
 'rest_framework',
 'taggit',
 'django_select2',
 'qrcode',
 'multiselectfield',
 'crispy_forms',
 'django_nyt',
 'mptt',
 'sorl.thumbnail',
 'easy_thumbnails',
 'sekizai',
 'wiki',
 'wiki.plugins.attachments',
 'wiki.plugins.images')
Installed Middleware:
('django.middleware.common.CommonMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'VTK.apps.centralauth.middleware.CentralAuthMiddleware',
 'django.middleware.common.BrokenLinkEmailsMiddleware',
 'django.middleware.locale.LocaleMiddleware',
 'django.contrib.flatpages.middleware.FlatpageFallbackMiddleware',
 'django.contrib.redirects.middleware.RedirectFallbackMiddleware')



Traceback:

File "/home/nico/.virtualenvs/vtk/lib/python3.5/site-packages/django/core/handlers/exception.py" in inner
  39.             response = get_response(request)

File "/home/nico/.virtualenvs/vtk/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
  187.                 response = self.process_exception_by_middleware(e, request)

File "/home/nico/.virtualenvs/vtk/lib/python3.5/site-packages/django/core/handlers/base.py" in _get_response
  185.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/home/nico/.virtualenvs/vtk/lib/python3.5/site-packages/django/contrib/admin/options.py" in wrapper
  544.                 return self.admin_site.admin_view(view)(*args, **kwargs)

File "/home/nico/.virtualenvs/vtk/lib/python3.5/site-packages/django/utils/decorators.py" in _wrapped_view
  149.                     response = view_func(request, *args, **kwargs)

File "/home/nico/.virtualenvs/vtk/lib/python3.5/site-packages/django/views/decorators/cache.py" in _wrapped_view_func
  57.         response = view_func(request, *args, **kwargs)

File "/home/nico/.virtualenvs/vtk/lib/python3.5/site-packages/django/contrib/admin/sites.py" in inner
  211.             return view(request, *args, **kwargs)

File "/home/nico/.virtualenvs/vtk/lib/python3.5/site-packages/django/utils/decorators.py" in _wrapper
  67.             return bound_func(*args, **kwargs)

File "/home/nico/.virtualenvs/vtk/lib/python3.5/site-packages/django/utils/decorators.py" in _wrapped_view
  149.                     response = view_func(request, *args, **kwargs)

File "/home/nico/.virtualenvs/vtk/lib/python3.5/site-packages/django/utils/decorators.py" in bound_func
  63.                 return func.__get__(self, type(self))(*args2, **kwargs2)

File "/home/nico/.virtualenvs/vtk/lib/python3.5/site-packages/django/contrib/admin/options.py" in changelist_view
  1585.                 response = self.response_action(request, queryset=cl.get_queryset(request))

File "/home/nico/.virtualenvs/vtk/lib/python3.5/site-packages/django/contrib/admin/options.py" in response_action
  1305.             response = func(self, request, queryset)

File "/home/nico/.virtualenvs/vtk/lib/python3.5/site-packages/mptt/admin.py" in delete_selected_tree
  84.                         obj_display = force_text(obj)

File "/home/nico/.virtualenvs/vtk/lib/python3.5/site-packages/django/utils/encoding.py" in force_text
  76.                     s = six.text_type(s)

File "/home/nico/.virtualenvs/vtk/lib/python3.5/site-packages/wiki/models/urlpath.py" in __str__
  192.         path = self.path

File "/home/nico/.virtualenvs/vtk/lib/python3.5/site-packages/wiki/models/urlpath.py" in path
  133.                 self.cached_ancestors))

File "/home/nico/.virtualenvs/vtk/lib/python3.5/site-packages/wiki/models/urlpath.py" in __cached_ancestors
  107.         if not self.get_ancestors().exists():

File "/home/nico/.virtualenvs/vtk/lib/python3.5/site-packages/mptt/models.py" in _fn
  371.                 % {'function': func.__name__, 'class': self.__class__.__name__}

Exception Type: ValueError at /admin/wiki/urlpath/
Exception Value: Cannot call get_ancestors on unsaved URLPath instances
```

**Comment by benjaoming** [comment]
Fixed in #683
